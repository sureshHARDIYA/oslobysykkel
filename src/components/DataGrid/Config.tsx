export const DefaultColumns = {
  width: 250,
  resizable: true,
  sortable: true,
  filter: 'agTextColumnFilter',
  floatingFilter: true,
};

export const ColumnNames = [
  {
    field: 'station_id',
    headerName: 'Station ID',
    width: 200,
  },
  {
    field: 'name',
    headerName: 'Station Name',
  },
  {
    field: 'address',
    headerName: 'Address',
  },
  {
    field: 'lat',
    headerName: 'Latitude',
    filter: 'agNumberColumnFilter',
  },
  {
    field: 'lon',
    headerName: 'Longitude',
    filter: 'agNumberColumnFilter',
  },
  {
    field: 'num_bikes_available',
    headerName: 'Number of available bikes',
    filter: 'agNumberColumnFilter',
  },
  {
    field: 'num_docks_available',
    headerName: 'Number of available docks',
    filter: 'agNumberColumnFilter',
  },
  {
    field: 'capacity',
    headerName: 'Capacity',
    filter: 'agNumberColumnFilter',
    width: 150,
  },
  {
    field: 'is_renting',
    headerName: 'Is renting?',
  },
];
