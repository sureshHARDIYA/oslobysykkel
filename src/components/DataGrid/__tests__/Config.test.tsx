import { DefaultColumns, ColumnNames } from '../Config';

describe('Config', () => {
  it('should match the snapshot for DefaultColumns', () => {
    expect(DefaultColumns).toMatchSnapshot();
  });

  it('should match the snapshot for ColumnNames', () => {
    expect(ColumnNames).toMatchSnapshot();
  });
});
