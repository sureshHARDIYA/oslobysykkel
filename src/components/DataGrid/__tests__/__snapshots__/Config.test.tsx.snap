// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Config should match the snapshot for ColumnNames 1`] = `
Array [
  Object {
    "field": "station_id",
    "headerName": "Station ID",
    "width": 200,
  },
  Object {
    "field": "name",
    "filter": "agSetColumnFilter",
    "headerName": "Station Name",
  },
  Object {
    "field": "address",
    "headerName": "Address",
  },
  Object {
    "field": "num_bikes_available",
    "filter": "agNumberColumnFilter",
    "headerName": "Number of available bikes",
  },
  Object {
    "field": "num_docks_available",
    "filter": "agNumberColumnFilter",
    "headerName": "Number of available docks",
  },
  Object {
    "field": "capacity",
    "filter": "agNumberColumnFilter",
    "headerName": "Capacity",
    "width": 150,
  },
  Object {
    "field": "is_renting",
    "headerName": "Is renting?",
  },
]
`;

exports[`Config should match the snapshot for DefaultColumns 1`] = `
Object {
  "filter": "agTextColumnFilter",
  "floatingFilter": true,
  "resizable": true,
  "sortable": true,
  "width": 250,
}
`;
